qa/rpc-tests/maxuploadtarget.py

```diff
@@ -175,13 +175,13 @@ def run_test(self):
         getdata_request = msg_getdata()
         getdata_request.inv.append(CInv(2, big_old_block))
 
-        max_bytes_per_day = 200*1024*1024
-        daily_buffer = 144 * MAX_BLOCK_SIZE
+        max_bytes_per_day = 800*1024*1024
+        daily_buffer = 144 * 4000000
```

Comment:
Oh, I misunderstood your comment. Sure, I should add a constant for the new max block size on the python side.